camel/models/openai_model.py

```diff
@@ -93,6 +93,20 @@ def run(
                 `ChatCompletion` in the non-stream mode, or
                 `Stream[ChatCompletionChunk]` in the stream mode.
         """
+        if self.model_type in [ModelType.O1_MINI, ModelType.O1_PREVIEW]:
```

Comment:
```suggestion
        """
        # o1-preview and o1-mini have Beta limitations 
        # reference: https://platform.openai.com/docs/guides/reasoning
        if self.model_type in [ModelType.O1_MINI, ModelType.O1_PREVIEW]:
```

Reply:
In fact, in all LLM models, including GPT, the system message is optional. The model can work without the system message. This is why I modified ChatAgent.

Reply:
I think it's important to define agent using system message, the change for `ChatAgent` need further discussion, how about we just keep the change focusing on model side and fix the bug first?